ts/Extensions/PatternFill.ts

```diff
@@ -223,6 +224,15 @@ Point.prototype.calculatePatternDimensions = function (
         if (!bBox.width || !bBox.height) {
             pattern._width = 'defer';
             pattern._height = 'defer';
+
+            // Check if map image pattern is flipped (#16810)
```

Comment:
This comment here is repeated in 2 different places but both parts of code are different so I think one of the comments should be changed or should be more descriptive